README.md

```diff
@@ -44,24 +44,38 @@ Fly is extremely extensible, so _anything_ can be a task. Our core system will a
 <h2>Table of Contents</h2>
 <details>
 <summary>Table of Contents</summary>
-- [Features](#features)
```

Comment:
No, I mean just add new line on line `47` without rewriting structure to tags and it will fix the problem.